include/PinPatternManager.h

```diff
@@ -11,7 +11,7 @@
 namespace OpenShock {
   class PinPatternManager {
   public:
-    PinPatternManager(unsigned int gpioPin);
+    PinPatternManager(std::uint8_t gpioPin);
```

Comment:
With this change, its not wrong or anything but we need to do checks before calling this class to make sure the integer is less than 256.
Dunno if this is already in place 🤔

Reply:
Same goes for the RFTransmitter class. Is there a change you were looking for, or just making the observation?

Reply:
Some input validation would be nice, but i fixed that so looks good to me :)